src/components/shared/ModalRegistration/AuthForm/index.tsx

```diff
@@ -28,15 +29,16 @@ const AuthForm: FC<AuthProps> = ({ isModalOpen, setIsModalOpen }) => {
 
     dispatch(thunkAuthSignin({ email: login, password })).then(res => {
```

Comment:
![image](https://github.com/kosta4310/farmers-market/assets/86911989/6e9a780c-dc12-4319-bcae-e79730e08f67)
![image](https://github.com/kosta4310/farmers-market/assets/86911989/b6178ee3-c122-4b8e-a1ef-96562e1b5a76)
![image](https://github.com/kosta4310/farmers-market/assets/86911989/375322e1-c0b5-4e08-a2c5-1d0b469ec9dd)

Reply:
For me any errors, all works.